client/landing/jetpack-cloud/components/sidebar/index.jsx

```diff
@@ -0,0 +1,167 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import ExpandableSidebarMenu from 'layout/sidebar/expandable';
+import Sidebar from 'layout/sidebar';
+import SidebarFooter from 'layout/sidebar/footer';
+import SidebarItem from 'layout/sidebar/item';
+import SidebarMenu from 'layout/sidebar/menu';
+import SidebarRegion from 'layout/sidebar/region';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class JetpackCloudSidebar extends Component {
+	static propTypes = {
+		path: PropTypes.string.isRequired,
+	};
+
+	state = {
+		expandedSections: this.props.path.match( /^(\/?[^/]*)/gi ), // Get the first segment, e.g. `/backups`
```

Comment:
I might have said that before, but ideally, this selection should be handled by the user preference. Just so it would be preserved after a refresh. Maybe let's add a TODO here and we can leave it for another time.

Reply:
That's a good idea. Since we're not lifting the state to the store, how about using a [browser storage](https://github.com/Automattic/wp-calypso/blob/81162c9e9c8c2be8e78cac04e4c8c8735963434a/client/lib/browser-storage/) here? In Calypso it's used to store the Redux store, so I'm not sure if using it just for storing this particular state would be the best way to go?

For now I'm going to add a todo comment here.

Reply:
I think that in order to have the sections state persisted we will have to eventually lift it to Redux, as it is in Calypso right now. This way it'll work "out of the box".

A todo comment is now added so I think that we can leave it as is until #38977 is merged.